frontend/src/components/LandingPage/Weather/Weather.js

```diff
@@ -2,9 +2,9 @@ import React from 'react';
 import './Weather.css'
 
 const weatherData = {
-    city: "Allen" + " ",
-    temperature: "65" + '\xB0F ',
-    picture: "/partly cloudy.png"
+    city: "Allen ",
+    temperature: "65 ",
```

Comment:
To make sure there is no confusion in temperature, I suggest labeling the temperature as Fahrenheit or Celsius.   `temperature: "65\xB0F ",`